Docs/source/usage/parameters.rst

```diff
@@ -2395,6 +2395,39 @@ Reduced Diagnostics
 
         Note that the fields are averaged on the cell centers before the reduction is performed.
 
+    * ``RawEFieldReduction``
+        This type is ONLY for the E-field at the respected staggering and executes the parser 
+        as a function of (x,y,z) to reduce Ex, Ey, and Ez.
+
+        * ``<reduced_diags_name>.reduction_type`` (`string`)
+            The type of reduction to be performed. It must be either ``Maximum``, ``Minimum`` or
+            ``Integral``.
+            ``Integral`` computes the spatial surface or volume integral, depending on the choice
+            of the ``integration_type``, of the function defined in the parser by summing its value on
+            all grid points and multiplying the result by the area or volume of a cell.
+            Please be also aware that measuring maximum quantities might be very noisy in PIC
+            simulations.
+
+        The output columns correspond to the timestep counter, physical time, and reduced values of Ex, Ey, Ez components.
+
+        * ``<reduced_diags_name>.integration_type`` (`string`)
+           The type of integration to be performed. It must be either ``surface`` or ``volume``.
+           Note that the surface integral provides the integrated normal as well as the
+           traverse fields over the surface. For the surface integral, the user-defined parser
+           for the plane should be less than one cell size in thickness in the direction of the surface normal.
+           Also, if the specified surface has an edge that overlaps with the domain boundary but the edge parallel
+           to it does not overlap with the domain boundary, then exclude the edge that overlaps with the domain boundary
+           while defining the surface.
+           See examples in Examples/Tests/Macroscopic_Maxwell/inputs_3d_surfaceIntegralDiagnostic_tests_cubic and
+           Examples/Tests/Macroscopic_Maxwell/inputs_3d_surfaceIntegralDiagnostic_tests_noncubic for ways to
+           define a full, half, and a quarter plane.
```

Comment:
```suggestion
```

if you want to given an example, typically, we write the example in the documentation of the specific part. 
Also, if for some reason, we decide to remove those tests, we may not remember to change the documentation accordingly :-)